dtm0/it/all2all/README.md

```diff
@@ -39,12 +53,13 @@ sudo ./all2all
 ## Hare usage
 
 Hare is used for cluster bootstrap and shutdown. Firstly
-it needs to be patched by the hare.patch in current directory
+it needs to be cloned locally and switched to dtm-0 branch
 to make it able to generate clusted configuration that includes
```

Comment:
s/clusted/cluster

Reply:
Fixed in https://github.com/Seagate/cortx-motr/pull/802/commits/72089222e7902508f02db886b882fd0dca3f6bca